packages/ui/src/components/SliderSteps/SliderSteps.test.tsx

```diff
@@ -1,15 +1,79 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
+import React, { useState } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
 
 import { SliderSteps } from './SliderSteps';
 
 describe('SliderSteps', () => {
+  const givenStepsCount = 5;
+  const lastDotIndex = givenStepsCount - 1;
+  const selectedIndex = 2;
+
+  const TestSliderComponent = ({ startSelectedIndex }: { startSelectedIndex: number }) => {
+    const [selected, setSelectedIndex] = useState(startSelectedIndex);
+
+    return <SliderSteps count={givenStepsCount} selectedIndex={selected} onChange={setSelectedIndex} />;
+  };
+
+  const getSelectedDotIndex = () =>
+    screen.getAllByRole('tab').findIndex((x) => x.getAttribute('aria-selected') === 'true');
+
+  const fireButtonClick = (buttonLabel: string) => {
+    const button = screen.getByLabelText(buttonLabel);
+
+    fireEvent.click(button);
```

Comment:
💭 You can use the [user-event](https://github.com/testing-library/user-event) library. It will be a small improvement as it uses events that are more similar to the one's users cause.

Reply:
Using this library is good idea in general, but in that case it will change only 
```diff
    fireEvent.click(button);
```
to
```diff
    userEvent.click(button);
```
(or maybe I don't know about sth?) 
and what's more, if I implement your suggestion about testing only the component - not its implementation, this function become useless and ready to remove

Reply:
It's more about a standard than the number of use cases. When someone will implement a similar test in the future he/she could look at your code and copy the solution. If we agree that using `userEvent` is a better approach than `fireEvent` we should start using it from the first suitable use case.

I don't understand why this function becomes useless when you apply my suggestion about testing `onChange`. You would still need to click something and check arguments of the called `onChange` function.